config/locales/en.yml

```
@@ -967,6 +967,16 @@ en:
       default: Selection from a list
       none_of_the_above: None of the above
       only_one_option: Selection from a list, one option only.
+  page_route_card:
+    check_your_answers: Check your answers before submitting
+    conditional_answer_value: '"%{answer_value}"'
+    conditional_route_title: Route %{index}
+    continue_to: Continue to
+    default_route_title: For any other answer
+    if_answer_is: If the answer is
+    page_name: "%{page_position}. %{page_name}"
```

```suggestion
    page_name: %{page_position}. %{page_name}
```
Hoping this removes the double quotes around the question at the top of the summary card?